custom_components/tuya_local/devices/neatsvor_x520ru_vacuum.yaml

```diff
@@ -0,0 +1,153 @@
+name: neatsvor_x520_vacuum_cleaner
+products:
+  - id: dbougox8k7ntyxeh
+    name: Neatsvor X520RU
+    manufacturer: Neatsvor
+    model: X520RU
+entities:
+  - entity: vacuum
+    dps:
+      - id: 1
+        type: boolean
+        name: power
+      - id: 2
+        type: boolean
+        name: activate
+      - id: 3
+        type: string
+        name: command
+        mapping:
+          - dps_val: "standby"
+            value: "standby"
+          - dps_val: "smart"
+            value: "smart"
+          - dps_val: "wall_follow"
+            value: "wall_follow"
+          - dps_val: "mop"
+            value: "mop"
+          - dps_val: "spiral"
+            value: "clean_spot"
+          - dps_val: "chargego"
+            value: "return_to_base"
+      - id: 5
+        type: string
+        name: status
+        mapping:
+          - dps_val: "0"
+            value: "Standby"
```

Comment:
This needs to be all lower case to be picked up by the vacuum entity as a supported state.